spec/generators/install_generator_spec.rb

```diff
@@ -149,7 +149,7 @@
     end
 
     it "sorts models by dependency" do
-      expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
+      expect(content["models"].keys).to eq ["post", "simple_model", "user", "comment", "game_room", "game_room_user", "post_metadatum", "review"]
```

Comment:
This change is causing the tests to fail. Can you revert this?

Reply:
Reverted but when I run this test I'm getting this problem:
```Failures:

  1) Seedie::Generators::InstallGenerator#model_configuration sorts models by dependency
     Failure/Error: expect(content["models"].keys).to eq expect(content["models"].keys).to eq ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
     
       expected: ["user", "simple_model", "post", "review", "post_metadatum", "game_room", "game_room_user", "comment"]
            got: ["post", "simple_model", "user", "comment", "game_room", "game_room_user", "post_metadatum", "review"]
     
       (compared using ==)
     # ./spec/generators/install_generator_spec.rb:152:in `block (3 levels) in <top (required)>'

Finished in 1.61 seconds (files took 0.93248 seconds to load)
160 examples, 1 failure

Failed examples:

rspec ./spec/generators/install_generator_spec.rb:151 # Seedie::Generators::InstallGenerator#model_configuration sorts models by dependency```

Reply:
Interesting, maybe your db has a different schema, will have to take a deeper look.